math/sympol/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               boost 1.0
+PortGroup               cmake 1.1
+PortGroup               github 1.0
+
+github.setup            tremlin SymPol 0.1.9 v
+name                    sympol
+revision                0
+categories              math science
+license                 GPL2
+maintainers             {@catap korins.ky:kirill} openmaintainer
+description             C++ tool to work with symmetric polyhedra.
+long_description        {*}${description} It helps to compute restricted automorphisms \
+    (parts of the linear symmetry group) of polyhedra and performs polyhedral description \
+    conversion up to a given or computed symmetry group.
+
+homepage                http://www.math.uni-rostock.de/~rehn/software/sympol.html
+
+checksums               rmd160  5fce5bbcb8ef1b521c39ef2d2cd5f334818c7736 \
+                        sha256  88e32321fa98eaf614abfda9be7fad992282db37952604707e79939f1144dd5a \
+                        size    234244
+
+patchfiles-append       patch-bliss-0.77.diff \
+                        patch-cddlib-0.94k.diff \
+                        patch-lrs-071.diff \
+                        patch-sympol-fix-headers.diff \
+                        patch-modern-compilers.diff
```

Comment:
please add links as to indicate where these patches come from; if not from upstream then the question becomes: are they submitted/accepted upstream?

Reply:
@reneeotten upstream is long dead. Well used math library usually is written as software for someones PhD thesis :) Rarely it is supported. Unfortunately this isn't an exception.

This patches is based on patches fedora and arch linux, but it requiers some work to be migrated to clang and used version of dependencies in MacPorts.

I really doubt that anyone will touch it because it wasn't updated for years.

Reply:
Anyway, I've added some notes inside patches.